telemetry/istio/src/main/scala/io/buoyant/telemetry/istio/MixerClient.scala

```diff
@@ -0,0 +1,114 @@
+package io.buoyant.telemetry.istio
+
+import com.google.local.DurationProto.Duration
+import com.twitter.finagle.buoyant.h2
+import com.twitter.finagle.Service
+import com.twitter.util.Future
+import istio.mixer.v1.{Attributes, Mixer, ReportRequest, ReportResponse, StringMap}
+
+private[telemetry] object MixerClient {
+
+  // TODO: build up this dictionary as requests are sent out,
+  // also consider using defaultWords, once per ReportRequest, rather than per Attributes
+  private[telemetry] val words = Seq[String](
```

Comment:
Thanks for the feedback @ZackButcher ! I didn't realize the global dictionary was embedded in Mixer, we'll remove this duplication in that case. We may still end up reading/importing `global_dictionary.yaml` since we'll need to reference those strings in `Attributes`, but good to know we don't need to send them over the wire.